### PR TITLE
Added MessageGUID to the output of Proofpoint TAP

### PR DESCRIPTION
**Pull request by tylerslijboom-r7**
## Proposed Changes

Added 1 optional field to the "message" output object, "message_guid" that is extracted from the TAP alert.

### Description

See above

### Testing

icon-plugin -R tests/testing.json parses out the field.

### CODE REVIEW

File: proofpoint_tap/plugin.spec.yaml [hunk 1]
```diff
@@ -78,6 +78,11 @@ types:
       description: Header reply to
       type: string
       required: false
+    message_guid:
```
Comment (mrinehart-r7) [inline proofpoint_tap/plugin.spec.yaml:81]: I don't see this used in code anywhere?

Comment (tylerslijboom-r7) [reply proofpoint_tap/plugin.spec.yaml:81]: proofpoint_tap/komand_proofpoint_tap/util/tap_formatter.py  has it added.  The value will be extracted from the HTML just like the other values in that clean_data hash in tap_formatter.py

**Comment by tylerslijboom-r7** [comment]
> There's a Threat-ID field that we need to extract too.

Added and updated the branch.

**Comment by jschipp-r7** [comment]
Action verified as working

**Pull request merged**